### PR TITLE
add moderation email to managewiki settings

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -281,6 +281,19 @@
 		'help' => 'If enabled, moderators are allowed to edit pending changes before approving. DANGEROUS: moderator can accidentally delete the text of pending change. Enable this only when you use Moderation for pre-publish review.',
 		'requires' => [],
 	],
+	'wgModerationEmail' => [
+		'name' => 'Moderation Email',
+		'from' => 'moderation',
+		'type' => 'text',
+		'overridedefault' => false,
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:288]:
```suggestion
		'overridedefault' => $wgPasswordSender,
```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Review by Universal-Omega — APPROVED** [review]

File: ManageWikiSettings.php [hunk 2]
```diff
@@ -281,6 +281,19 @@
 		'help' => 'If enabled, moderators are allowed to edit pending changes before approving. DANGEROUS: moderator can accidentally delete the text of pending change. Enable this only when you use Moderation for pre-publish review.',
 		'requires' => [],
 	],
+	'wgModerationEmail' => [
+		'name' => 'Moderation Email',
+		'from' => 'moderation',
+		'type' => 'text',
+		'overridedefault' => $wgPasswordSender,
+		'section' => 'anti-spam',
+		'help' => 'If enabled, moderators are allowed to edit pending changes before approving. DANGEROUS: moderator can accidentally delete the text of pending change. Enable this only when you use Moderation for pre-publish review.',
```
Comment (edwardspec) [inline ManageWikiSettings.php:290]: copy-paste error (this is description of the `$wgModerationEnableEditChange` above)

**Comment by RhinosF1** [comment]
@Reception123 don'T believe this will work. It will still use our mail servers which will reject it.

File: ManageWikiSettings.php (hunk 2 above)
Comment (Universal-Omega) [reply ManageWikiSettings.php]: Fixed that. Thanks!

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**